tests/ignite/engine/test_create_supervised.py

```diff
@@ -460,23 +460,35 @@ def test_create_supervised_trainer_on_cuda_with_model_on_cpu():
 def test_create_supervised_evaluator():
     _test_create_supervised_evaluator()
     _test_mocked_supervised_evaluator()
-    with mock.patch("torch.cuda.amp.autocast") as mock_torch_cuda_amp_module:
-        _test_create_evaluation_step_amp(mock_torch_cuda_amp_module)
+    try:
```

Comment:
Maybe we could simply skip those subtests with [pytest.skip](https://docs.pytest.org/en/stable/skipping.html#skipping-test-functions) similarly to `pytest.mark.skipif(LooseVersion(torch.__version__) < LooseVersion("1.6.0"), reason="Skip if < 1.6.0")` (without decorator) ?
```python
def test_function():
    if not valid_config():
        pytest.skip("unsupported configuration")
```

Reply:
That's way cleaner actually, will do this now